processor/processor_test.go

```diff
@@ -635,7 +635,7 @@ var _ = Describe("Processor with event schemas v2", Ordered, func() {
 				StoreInTx(gomock.Any(), gomock.Any(), gomock.Any()).
 				Times(1).
 				Do(func(ctx context.Context, tx jobsdb.StoreSafeTx, jobs []*jobsdb.JobT) {
-					Expect(jobs).To(HaveLen(2))
+					Expect(jobs).To(HaveLen(5))
```

Comment:
Earlier we were reading the `job_run_id` from the body and now we are doing the same from params. So none of the params have these set and this resulted in increase of jobs

Reply:
Can we modify test data so that test behaviour remains the same? It would be valuable to verify that event schemas v2 skips events from rudder-sources, correct?